jac-uk/admin
# #2099 QT Merit list. Include did not take and failed first test candidates

## What's included?
Closes #2099 

Following feedback from Sean (stats guy) we need to make the following improvements:

 - [x] Include candidates who did not take both the CAT and SJT tests. These candidates will be eventually be assigned a status of `noTestSubmitted`
 - [x] Include candidates who took one or both tests but failed one or both of them. These candidates will eventually be assigned a status of 'failedFirstTest'
 - [x] Include the following columns of data (in this order with matching column names):
   - Reference number
   - Full name
   - Email
   - SJT score
   - SJT %
   - CAT score
   - CAT %
   - Z_SJT
   - Z_CAT
   - Z_Overall
   - Rank
   - Notes
   - Outcome
- [x] Note 'average Z score' is now 'overall z score', due to the change from 50/50 weighting to 60/40. Check on screen wording supports this

## Who should test?
✅ Product owner
✅ Developers

## How to test?
These changes arose from discussions with SET running 162 recorder. That team have tested and are happy

If we are quick then the download can be seen here (note this is a live exercise!):
https://jac-apply-admin-production--pr2100-improvement-2099-r7vruyc8.web.app/exercise/BpEL3HgHEyHCvDMI5E9s/tasks/qualifyingTest/finalised

Press the Export button and download the full data
![image.png](https://images.zenhubusercontent.com/5e412648a39708a606c962c9/accef8ff-d13d-4e2b-b7a7-4813a6c22df7)

At the end of the spreadsheet you should see rows for 'noTestSubmitted' and 'failedFirstTest'.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:PRODUCTION
_can be OFF, DEVELOP or STAGING_
